app/packages/looker/src/lookers/imavid/buffer-manager/buffer-manager.test.ts

```diff
@@ -0,0 +1,242 @@
+import { beforeEach, describe, expect, test } from "vitest";
```

Comment:
Awesome! These tests are super valuable. It'd be cool to apply the buffer manager to `VideoLooker` in the future 🚀

Reply:
100%